### PR TITLE
Add Flag for Due and  Overdue for third DNA PCR

### PR DESCRIPTION
**Pull request by solemabrothers**
The Patient Flag for clients who are overdue for 3rd DNA PCR https://app.asana.com/0/1133167201254915/1160475826379202

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/UgandaEMRPatientFlagMetadataBundle.java [hunk 1]
```diff
@@ -43,6 +44,7 @@ public void install() throws Exception {
         install(Flags.DUE_FOR_SECOND_DNA_PCR);
         install(Flags.OVERDUE_FOR_SECOND_DNA_PCR);
         install(Flags.DUE_FOR_RAPID_TEST);
+        install(Flags.OVER_DUE_FOR_THIRD_DNA_PCR);
```
Comment (ebichete) [inline api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/UgandaEMRPatientFlagMetadataBundle.java:48]: Overdue is one word. Follow the convention is in use on "Rapid Test", "First DNA PCR" and "Second DNA PCR".

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java [hunk 2]
```diff
@@ -532,6 +532,53 @@ public String uuid() {
...
+                    "AND p.patient_id NOT IN (SELECT ee.patient_id FROM encounter ee INNER JOIN encounter_type ete ON ee.encounter_type = ete.encounter_type_id\n" +
+                    "                       WHERE ete.uuid = '8d5b27bc-c2cc-11de-8d13-0010c6dffd0f' AND ee.voided = FALSE)\n" +
+                    "AND oo.voided = FALSE   group by oo.person_id\n" +
+                    "  HAVING TIMESTAMPDIFF(WEEK,MIN(oo.obs_datetime),CURDATE())>=7  and  p.patient_id  IN (SELECT oo.person_id FROM obs oo WHERE oo.concept_id = 99436 AND oo.voided = FALSE)\n" +
+                    "AND p.patient_id NOT IN (SELECT oo.person_id FROM obs oo WHERE oo.concept_id = 90306 AND oo.voided = FALSE)";
+        }
+
+        @Override
+        public String message() {
+            return "Over Due for 3rd DNA PCR on ${1}";
```
Comment (ebichete) [inline api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java:597]: Overdue is one word. Follow the convention is in use on "Rapid Test", "First DNA PCR" and "Second DNA PCR".

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java [hunk 3]
```diff
@@ -532,6 +532,53 @@ public String uuid() {
...
+        }
+
+        @Override
+        public List<String> tags() {
+            return Arrays.asList(Tags.PATIENT_STATUS.uuid());
+        }
+
+        @Override
+        public String name() {
+            return "Over Due for 3rd DNA PCR";
```
Comment (ebichete) [inline api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java:612]: Overdue is one word. Follow the convention is in use on "Rapid Test", "First DNA PCR" and "Second DNA PCR".

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java [hunk 4]
```diff
@@ -532,6 +532,53 @@ public String uuid() {
...
+        }
+
+        @Override
+        public String description() {
+            return "Exposed infants who are over due for their third DNA PCR, 6 weeks after breastfeeding and having taken a 2nd DNA PCR";
```
Comment (ebichete) [inline api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java:617]: Overdue is one word. Follow the convention is in use on "Rapid Test", "First DNA PCR" and "Second DNA PCR".

File: api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/UgandaEMRPatientFlagMetadataBundle.java (hunk 1 above)
Comment (solemabrothers) [reply api/src/main/java/org/openmrs/module/aijar/api/deploy/bundle/UgandaEMRPatientFlagMetadataBundle.java]: Done

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java (hunk 2 above)
Comment (solemabrothers) [reply api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java]: Done

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java (hunk 3 above)
Comment (solemabrothers) [reply api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java]: Done

File: api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java (hunk 4 above)
Comment (solemabrothers) [reply api/src/main/java/org/openmrs/module/aijar/metadata/core/Flags.java]: Done

**Pull request merged**